app/models/ability.rb

```diff
@@ -23,7 +23,8 @@ def initialize(user)
       can :manage, Availability, speaker_id: user.id
     else
       can :update, User, id: user.id
-      can :profile, User, id: user.id
+      can :profile, User
```

Comment:
should it rather be like this?
```suggestion
      can :profile, User, id: user.id
```

Reply:
I would like them to see other people's profiles too so that they can view the speaker profile when they click the button on the speaker index page. If I put user can see a profile only if it matches his id, then he can't look at a speaker profile, right?

Reply:
Okay